ios/FluentUI.Demo/FluentUI.Demo/Demos/SegmentedControlDemoController.swift

```diff
@@ -91,4 +87,101 @@ class SegmentedControlDemoController: DemoController {
             controlLabels[pillControl] = addDescription(text: "", textAlignment: .center)
         }
     }
+
+    private var segmentedControls: [SegmentedControl] = []
+}
+
+extension SegmentedControlDemoController: DemoAppearanceDelegate {
+    func themeWideOverrideDidChange(isOverrideEnabled: Bool) {
+        guard let fluentTheme = self.view.window?.fluentTheme else {
+            return
+        }
+
+        var tokensClosure: ((SegmentedControl) -> SegmentedControlTokens)?
+        if isOverrideEnabled {
+            tokensClosure = { _ in
+                return ThemeWideOverrideSegmentedControlTokens()
+            }
+        }
+
+        fluentTheme.register(controlType: SegmentedControl.self, tokens: tokensClosure)
+    }
+
+    func perControlOverrideDidChange(isOverrideEnabled: Bool) {
+        self.segmentedControls.forEach({ segmentedControl in
+            let tokens = isOverrideEnabled ? PerControlOverrideSegmentedControlTokens() : nil
+            _ = segmentedControl.overrideTokens(tokens)
+        })
+    }
+
+    func isThemeWideOverrideApplied() -> Bool {
+        return self.view.window?.fluentTheme.tokenOverride(for: SegmentedControl.self) != nil
+    }
+
+    // MARK: - Custom tokens
+
+    private class ThemeWideOverrideSegmentedControlTokens: SegmentedControlTokens {
+        override var font: FontInfo {
+            return FontInfo(name: "Times", size: 20.0, weight: .regular)
```

Comment:
can we not passin the size so even with Times font the accessibility text will still layout correctly?

Reply:
FontInfo needs all three.

Reply:
I updated the font to not scale in the SegmentPillButton